chart/kubeapps/values.yaml

```diff
@@ -1626,7 +1626,7 @@ kubeappsapis:
     pullSecrets: []
   ## @param kubeappsapis.replicaCount Number of frontend replicas to deploy
   ##
-  replicaCount: 1
+  replicaCount: 2
```

Comment:
Thanks, this should have been updated to 2 once we started using the kubeapps-apis in released code. (Or even better, I should have made it 2 from the start even when we weren't using it since it wasn't deployed in that case, zero cost without having to remember).